package.json

```diff
@@ -9,7 +9,8 @@
   "scripts": {
     "watch": "webpack --config ./webpack.config.js --watch",
     "build": "webpack --config ./webpack.config.js",
-    "test": "karma start"
+    "test": "karma start",
+    "lint": "eslint normandy client webpack.config.js karam.conf.js"
```

Comment:
one annoying thing about this is when there is a lint error. npm errors out too and spits out all of this:
```
npm ERR! Linux 3.13.0-79-generic
npm ERR! argv "/home/rehandalal/.nvm/versions/node/v4.0.0/bin/node" "/home/rehandalal/.nvm/versions/node/v4.0.0/bin/npm" "run" "lint"
npm ERR! node v4.0.0
npm ERR! npm  v2.14.2
npm ERR! code ELIFECYCLE
npm ERR! normandy@ lint: `eslint normandy client webpack.config.js karam.conf.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the normandy@ lint script 'eslint normandy client webpack.config.js karam.conf.js'.
npm ERR! This is most likely a problem with the normandy package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     eslint normandy client webpack.config.js karam.conf.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls normandy
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/rehandalal/Projects/normandy/npm-debug.log
```

and creates the file `npm-debug.log`.